autopush/settings.py

```diff
@@ -341,6 +342,29 @@ def make_endpoint(self, uaid, chid, key=None):
         ep = self.fernet.encrypt(base + sha256(raw_key).digest()).strip('=')
         return root + 'v2/' + ep
 
+    def extract_auth(self, auth_header):
+        vapid_auth = {}
```

Comment:
Maybe this should be an attrs style object (like WebPushNotification), would help ensure proper dict access to the few fixed keys here, and the logic could be better separated from settings.